pkg/apis/core/types.go

```diff
@@ -3000,14 +3000,12 @@ type PodSpec struct {
 	// - spec.containers[*].securityContext.runAsGroup
 	// +optional
 	OS *PodOS
-	// Each scheduling gate represents a particular scenario the scheduling is blocked upon.
-	// Scheduling is triggered only when SchedulingGates is empty.
+	// Each scheduling gate represents a particular scenario that must be satisfied before scheduling.
+	// Scheduling is triggered only when schedulingGates is empty.
 	// In the future, we may impose permission mechanics to restrict which controller can mutate
-	// which scheduling gate. It's dependent on a yet-to-be-implemented fined-grained
-	// permission (https://docs.google.com/document/d/11g9nnoRFcOoeNJDUGAWjlKthowEVM3YGrJA3gLzhpf4)
-	// and needs to be consistent with how finalizes/liens work today.
+	// which scheduling gate.
```

Comment:
The comment regarding the future is still there, I don't think this is useful in the api. Other fields point to the KEP, so why not just say: 

```
// More info: https://git.k8s.io/enhancements/keps/sig-scheduling/3521-pod-scheduling-readiness
```